tsconfig.dist.json

```diff
@@ -0,0 +1,21 @@
+{
```

Comment:
I don't understand why we need another `tsconfig`

Reply:
We need another `tsconfig` since declarations and `allowJs` are excluding each other and we need `allowJs` for storybook. And declarations for the dist.

Reply:
I pushed a commit addressing this one.